lib/v2/kakuyomu/episode.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const work_url = `https://kakuyomu.jp/works/${id}`;
+    const $ = cheerio.load(await get(work_url));
+
+    const work_title = $('#workTitle').text();
+    const introduction = $('#introduction').text();
+
+    const episode_list = $('a.widget-toc-episode-episodeTitle')
+        .slice(-5)
+        .map((_, episode) => {
+            const $_episode = $(episode);
+            return {
+                title: $_episode.find('span').text(),
+                link: $_episode.attr('href'),
```

Comment:
Please add pubDate
docs: https://docs.rsshub.app/joinus/pub-date.html
```diff
                title: $_episode.find('span').text(),
                link: $_episode.attr('href'),
+               pubDate: parseDate($_episode.find('time').attr('datetime'))
```

Reply:
kakuyomu 上作者可能修改之前的章节，因此章节顺序和更新时间不是严格对应的，比如[这本书](https://kakuyomu.jp/works/1177354054883783581)加上 pubDate 之后 rss 的显示顺序和 网页上的显示顺序是不一样的。

Reply:
I don't think even if the author updated previous chapters, the pubDate would be changed as the CSS class name said it's `datePublished`.

The order of RSS feed items doesn't align with the website because the author uploaded new chapters in bulk that new chapter is uploaded earlier than older chapters as you can see from the screenshot below.

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/11386903/159946603-72817fbe-dd9e-4163-8c81-a3eb9bffb7bf.png)
</details>

Apart from that, not all RSS software will follow the order shown in the RSS feed if no `pubDate` is provided. They may sort them based on the `title` or `link`, etc. That's why you should provide a `pubDate` to each feed item.